tests/lib/snaps/test-snapd-just-beta/snapcraft.yaml

```diff
@@ -0,0 +1,15 @@
+name: test-snapd-just-beta
+version: 1.0
+summary: Basic snap published just on beta channel
+description: A basic snap published just on beta channel
+confinement: strict
+grade: stable
```

Comment:
It would be much simpler to make this an: `architecture: [all]` snap but its fine.